integration-tests/bats/import-update-tables.bats

```diff
@@ -268,13 +288,19 @@ DELIM
 
     dolt sql < 1pk5col-ints-sch.sql
     dolt table import -u --continue test 1pk5col-rpt-ints.csv
+    dolt commit -am "cm1"
+
     run dolt table import -u --continue test 1pk5col-rpt-ints.csv
     [ "$status" -eq 0 ]
-    skip "Running this file on repeat produces modifications"
-    ! [[ "$output" =~ "Modifications: 2" ]] || falsa 
+    [[ "$output" =~ "Modifications: 3" ]] || falsa
+
+    run dolt diff
+    [ "$status" -eq 0 ]
+    [ "${#lines[@]}" -eq 0 ]
 }
 
 @test "import-update-tables: importing some columns does not overwrite columns not part of the import" {
+  skip "These types of partials should not be allowed. It is inconsistent with the mysql standard"
```

Comment:
@andrew-wm-arthur Can you verify this skip behavior. Disagree that is a valid test

Reply:
we need to preserve this behavior